test/acceptance/keywords/screenshots.robot

```diff
@@ -38,11 +39,42 @@ Capture page screenshot to custom root directory
     [Documentation]    Capture page screenshot to custom root directory
     [Setup]    Remove Directory    ${OUTPUTDIR}/custom-root    recursive
     Set Screenshot Directory    ${OUTPUTDIR}/custom-root
-    Capture Page Screenshot    custom-root-screenshot.png
+    ${file} =    Capture Page Screenshot    custom-root-screenshot.png
     File Should Exist    ${OUTPUTDIR}/custom-root/custom-root-screenshot.png
+    Should Be Equal    ${file}    ${OUTPUTDIR}${/}custom-root${/}custom-root-screenshot.png
 
 Ensure screenshot captures revert to default root directory
     [Documentation]    Ensure screenshot captures revert to default root directory
     [Setup]    Remove Files    ${OUTPUTDIR}/default-root-screenshot.png
     Capture Page Screenshot    default-root-screenshot.png
     File Should Exist    ${OUTPUTDIR}/default-root-screenshot.png
+
+Capture page screenshot with unique index
+    [Setup]    Remove Directory    ${OUTPUTDIR}${/}screenshot-and-index    recursive
+    ${file1} =    Capture Page Screenshot    ${OUTPUTDIR}${/}screenshot-and-index${/}other-{index}-name.png
+    ${file2} =    Capture Page Screenshot    ${OUTPUTDIR}${/}screenshot-and-index${/}some-other-name-{index}.png
+    ${file3} =    Capture Page Screenshot    ${OUTPUTDIR}${/}screenshot-and-index${/}other-{index}-name.png
+    File Should Exist    ${OUTPUTDIR}${/}screenshot-and-index${/}other-1-name.png
+    Should Be Equal    ${file1}    ${OUTPUTDIR}${/}screenshot-and-index${/}other-1-name.png
+    File Should Exist    ${OUTPUTDIR}${/}screenshot-and-index${/}some-other-name-1.png
+    Should Be Equal    ${file2}    ${OUTPUTDIR}${/}screenshot-and-index${/}some-other-name-1.png
+    File Should Exist    ${OUTPUTDIR}${/}screenshot-and-index${/}other-2-name.png
+    Should Be Equal    ${file3}    ${OUTPUTDIR}${/}screenshot-and-index${/}other-2-name.png
+
+Capturing a page screenshot with two indexes should not cause an error
+    ${file} =    Capture Page Screenshot    ${OUTPUTDIR}${/}screenshot-and-index${/}two-{index}-in-{index}-name.png
+    File Should Exist    ${OUTPUTDIR}${/}screenshot-and-index${/}two-1-in-1-name.png
+    Should Be Equal    ${file}    ${OUTPUTDIR}${/}screenshot-and-index${/}two-1-in-1-name.png
+
+Capture page screenshot whith index formatting
+    Capture Page Screenshot    ${OUTPUTDIR}${/}screenshot-and-index${/}format-{index:06}-name.png
+    Capture Page Screenshot    ${OUTPUTDIR}${/}screenshot-and-index${/}format-{index:06}-name.png
+    File Should Exist    ${OUTPUTDIR}${/}screenshot-and-index${/}format-000001-name.png
+    File Should Exist    ${OUTPUTDIR}${/}screenshot-and-index${/}format-000002-name.png
+
+Capture page screenshot with braces should now be doupled
```

Comment:
Perhaps rename this to `Capture page screenshot with escaped braces`. Current title doesn't make sense to me.